Documentation/HowToUseETW.md

```diff
@@ -0,0 +1,46 @@
+# Supported
```

Comment:
That's kind of a weird title. Consider making the first line title something like "Collecting and Analyzing ETW Traces" with "Support" being a subheading